libraries/standard/json/utest/CMakeLists.txt

```diff
@@ -0,0 +1,56 @@
+include("../jsonFilePaths.cmake")
+project ("json unit test")
+cmake_minimum_required (VERSION 3.13)
```

Comment:
(Unrelated to this PR) Are these macro calls about CMake versioning required in these unit test files? Should they instead exist in the master CMakeLists.txt file that includes these child Cmake files?

We need to also think from a broken repository perspective about how the minimum version would be enforced.

Reply:
I don't think this is necessary. We already have this as the minimum version in the root CMakeLists.txt file, so I will just remove this line and remove the rest in a separate PR.